ecl/hthor/hthor.cpp

```diff
@@ -235,11 +235,14 @@ void CHThorActivityBase::updateProgressForOther(IStatisticGatherer &progress, un
 
 void CHThorActivityBase::updateProgressForOther(IStatisticGatherer &progress, unsigned otherActivity, unsigned otherSubgraph, unsigned whichOutput, unsigned __int64 numProcessed) const
 {
-    StatsEdgeScope scope(progress, otherActivity, whichOutput);
-    progress.addStatistic(StNumRowsProcessed, numProcessed);
-    progress.addStatistic(StNumStarts, 1);  // wrong for an activity in a subquery
-    progress.addStatistic(StNumStops, 1);
-    progress.addStatistic(StNumSlaves, 1);  // MORE: A bit pointless for an hthor graph
+    if (numProcessed)
```

Comment:
@shamser This fix doesn't seem to match the bug.

This will not output results on an edge if an activity didn't process any records.  That is not the same - there are some example activities (e.g., PipeWrite) which do appear to increment processed, and it is valid for an activity to return 0 rows - and it should still report the statistics.